app/src/main/java/com/github/sdp/tarjetakuna/model/Coordinates.kt

```diff
@@ -15,7 +17,12 @@ data class Coordinates(
     }
 
     companion object {
-        const val latitudeRange = 90.0f
-        const val longitudeRange = 180.0f
+        const val latitudeRange = 90.0
+        const val longitudeRange = 180.0
+    }
+
+    fun isSameCoordAs(other: Coordinates): Boolean {
```

Comment:
I think it is better to override the function equals and not create a new one.

Reply:
I tried doing it but it requires to override hashcode as well and I have no idea how to do it (like what I should put in the hashcode )

Reply:
I think you can leave what Android Studio offers you, that is:

```
override fun hashCode(): Int {
    var result = latitude.hashCode()
    result = 31 * result + longitude.hashCode()
    return result
}
```